src/SourceGenerators/System.Xaml/System.Xaml/XamlXmlReader.cs

```diff
@@ -1039,12 +1039,12 @@ public IEnumerable<NamespaceDeclaration> GetNamespacePrefixes ()
 			}
 		}
 
-#if NET7_0_OR_GREATER && !DISABLE_GENERATED_REGEX
+#if NET7_0_OR_GREATER
 		[System.Text.RegularExpressions.GeneratedRegex("\\s+")]
 #endif
 		private static partial System.Text.RegularExpressions.Regex SpaceMatch();
 
-#if !NET7_0_OR_GREATER || DISABLE_GENERATED_REGEX
+#if !NET7_0_OR_GREATER
```

Comment:
Is this worth keeping now that we are .NET 7+?

Reply:
SGs are netstandard2.0